sdk/appconfiguration/azure-appconfiguration/HISTORY.md

```diff
@@ -3,6 +3,12 @@
 # Release History
 
 -------------------
+## 2019-10-xx Version 1.0.0b5
+
+### Breaking changes
+
+- delete_configuration_setting does not take etag and match_condition anymore (use kwargs instead)  #8161
```

Comment:
I would rephrase "etag and match_condition are now keyword argument only"

The way you say it "does not take [...] anymore" makes think there are not available anymore